### ISSUE TITLE
Input styles not following spec?

### ISSUE THREAD

**Issue by manuthecoder** [materializecss/materialize#241]
Hi again, 
Take a look at inputs from the Material design spec
![https://lh3.googleusercontent.com/FzKXH0mmW_btrkFxWJ-vo85kuzs1q-qjXbFXg0LWbMV7oW3kwd9xg9XllbEi4SjkLZR866LvdwE2UHux_YW5-T3239nWPxaSTXNkOQ=w1064-v0](https://lh3.googleusercontent.com/FzKXH0mmW_btrkFxWJ-vo85kuzs1q-qjXbFXg0LWbMV7oW3kwd9xg9XllbEi4SjkLZR866LvdwE2UHux_YW5-T3239nWPxaSTXNkOQ=w1064-v0)

In [this section](https://material.io/components/text-fields#research) it clearly mentions: 
> Use an enclosed text field with a rectangular box, rather than a single underline to indicate a text field
> Use either a semi-transparent fill with a bottom line or a fully transparent fill with an opaque stroke for the text field box

So, in conclusion, I think MaterializeCSS should include more styles of input fields. Hope I helped :)

**Comment by wuda-io** [comment]
implemented witth 2.X.X
